package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "cdmx009-mdlinks",
+  "version": "1.0.0",
+  "description": "## Preámbulo",
```

Te recomiendo que pueda complementar la información de la descripción (en ingles)
```suggestion
  "description": "",
```